src/content/blog/maintainer-job-description.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Recruiting DDEV Open Source Project Maintainers"
+pubDate: 2023-05-19
+summary: The DDEV community is seeking paid maintainers to boost the project's sustainability and growth.
+author: Randy Fay
+featureImage:
+    src: /public/img/blog/2023/05/diverse_group_of_people_sitting_at_a_table_working.png
```

Comment:
Fix path:

```suggestion
    src: /img/blog/2023/05/diverse_group_of_people_sitting_at_a_table_working.png
```

(I’ve also been using `-` as word separators for all Markdown and web-friendly URLs, but that’s a stylistic thing not called out anywhere.)

Reply:
Happy to do that, but why did it work? I thought I was just copying existing strategies in other articles.

Reply:
No idea! It didn’t work locally for me, which is the only reason I noticed.